### ISSUE TITLE
Alchemy Table spawns in an Indoor area at tribe; cannot be used due to lack of power

### ISSUE THREAD

**Issue by Girdo** [DesertRose2/desertrose#379]
## Round ID: 
3037

## Reproduction:

When clicking the alchemy table in the tribal area, it does not open the pill/bottling menu. Nothing happens at all. This is most likely because it spawns on a wooden 'indoor' floor- as an obj/machinery the alchemy table seems to be checking for an APC since it's in an indoor room, doesn't find one, and fails to function without power.
Creating a new one in the same place did not fix the issue, however when spawning an alchemy table one tile south. in an 'outdoor' tile the table worked as intended.

TL;DR Modifying the alchemy table to NOT require APC power would allow it to remain in its current spot, otherwise its spawn location will need to be moved to an outdoor area to retain permanent 'wasteland' power. Players are not able to create new tables/move on their own.
